docs/how-to/create-custom-image-using-existing-docker-image.md

```diff
@@ -0,0 +1,202 @@
+## How to create customized Docker images using an existing docker image.
+
+**WARNING**
+
+The recommended way of building extensions is to use [Rest APIs](https://api-explorer.alfresco.com/api-explorer). The AMP extension mechanism may be replaced in the future.
+
+#### Prerequisites
+
+* The extension is packaged inside an AMP
+* Docker
+* Access to Docker registry where the new Docker image will be pushed
+
+### Amps that don`t require additional configurations (easy).
```

Comment:
Change heading to: AMPs that don't require additional configuration (easy)
- Also remove the full stop.